From 0e5273a0afc8123f70ada3cad6f8ab6dd60dcded Mon Sep 17 00:00:00 2001
From: Calvin Johnson <calvin.johnson@nxp.com>
Date: Mon, 10 Dec 2018 10:03:51 +0530
Subject: [PATCH 423/741] arm64: dts: ls1012a: reorganize pfe_mac nodes

commit d6143699eefac304b718a725b0ddde234235daa6 from
http://source.codeaurora.org/external/qoriq/qoriq-components/linux

To keep platform specific properties in the platform dts files,
remove pfe_mac nodes from dtsi and define them in dts files.

Signed-off-by: Calvin Johnson <calvin.johnson@nxp.com>
Signed-off-by: Zhantao Tang <zhantao.tang@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-ls1012a-2g5rdb.dts | 4 ++--
 arch/arm64/boot/dts/freescale/fsl-ls1012a-frdm.dts   | 4 ++--
 arch/arm64/boot/dts/freescale/fsl-ls1012a-qds.dts    | 4 ++--
 arch/arm64/boot/dts/freescale/fsl-ls1012a-rdb.dts    | 4 ++--
 4 files changed, 8 insertions(+), 8 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-ls1012a-2g5rdb.dts b/arch/arm64/boot/dts/freescale/fsl-ls1012a-2g5rdb.dts
index 7bbfa1e886d4..119bf4b388c6 100644
--- a/arch/arm64/boot/dts/freescale/fsl-ls1012a-2g5rdb.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-ls1012a-2g5rdb.dts
@@ -52,7 +52,7 @@
 	#address-cells = <1>;
 	#size-cells = <0>;
 
-	ethernet@0 {
+	pfe_mac0: ethernet@0 {
 		compatible = "fsl,pfe-gemac-port";
 		#address-cells = <1>;
 		#size-cells = <0>;
@@ -62,7 +62,7 @@
 		phy-handle = <&sgmii_phy1>;
 	};
 
-	ethernet@1 {
+	pfe_mac1: ethernet@1 {
 		compatible = "fsl,pfe-gemac-port";
 		#address-cells = <1>;
 		#size-cells = <0>;
diff --git a/arch/arm64/boot/dts/freescale/fsl-ls1012a-frdm.dts b/arch/arm64/boot/dts/freescale/fsl-ls1012a-frdm.dts
index b6cdac49ffec..fcc3c271a4d2 100644
--- a/arch/arm64/boot/dts/freescale/fsl-ls1012a-frdm.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-ls1012a-frdm.dts
@@ -84,7 +84,7 @@
 	#address-cells = <1>;
 	#size-cells = <0>;
 
-	ethernet@0 {
+	pfe_mac0: ethernet@0 {
 		compatible = "fsl,pfe-gemac-port";
 		#address-cells = <1>;
 		#size-cells = <0>;
@@ -94,7 +94,7 @@
 		phy-handle = <&sgmii_phy1>;
 	};
 
-	ethernet@1 {
+	pfe_mac1: ethernet@1 {
 		compatible = "fsl,pfe-gemac-port";
 		#address-cells = <1>;
 		#size-cells = <0>;
diff --git a/arch/arm64/boot/dts/freescale/fsl-ls1012a-qds.dts b/arch/arm64/boot/dts/freescale/fsl-ls1012a-qds.dts
index 382df05ff813..a4fed83900ea 100644
--- a/arch/arm64/boot/dts/freescale/fsl-ls1012a-qds.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-ls1012a-qds.dts
@@ -142,7 +142,7 @@
 	#address-cells = <1>;
 	#size-cells = <0>;
 
-	ethernet@0 {
+	pfe_mac0: ethernet@0 {
 		compatible = "fsl,pfe-gemac-port";
 		#address-cells = <1>;
 		#size-cells = <0>;
@@ -152,7 +152,7 @@
 		phy-handle = <&sgmii_phy1>;
 	};
 
-	ethernet@1 {
+	pfe_mac1: ethernet@1 {
 		compatible = "fsl,pfe-gemac-port";
 		#address-cells = <1>;
 		#size-cells = <0>;
diff --git a/arch/arm64/boot/dts/freescale/fsl-ls1012a-rdb.dts b/arch/arm64/boot/dts/freescale/fsl-ls1012a-rdb.dts
index d7f13bc976c7..f112ea68f2be 100644
--- a/arch/arm64/boot/dts/freescale/fsl-ls1012a-rdb.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-ls1012a-rdb.dts
@@ -53,7 +53,7 @@
 	#address-cells = <1>;
 	#size-cells = <0>;
 
-	ethernet@0 {
+	pfe_mac0: ethernet@0 {
 		compatible = "fsl,pfe-gemac-port";
 		#address-cells = <1>;
 		#size-cells = <0>;
@@ -63,7 +63,7 @@
 		phy-handle = <&sgmii_phy>;
 	};
 
-	ethernet@1 {
+	pfe_mac1: ethernet@1 {
 		compatible = "fsl,pfe-gemac-port";
 		#address-cells = <1>;
 		#size-cells = <0>;
-- 
2.17.1

